modules/touch/src/touchinteraction.cpp

```diff
@@ -300,7 +313,8 @@ TouchInteraction::TouchInteraction()
     addProperty(_zoomSensitivityExponential);
     addProperty(_zoomSensitivityProportionalDist);
     addProperty(_zoomSensitivityDistanceThreshold);
-    addProperty(_zoomBoundarySphereMultiplier);
+    addProperty(_zoomInBoundarySphereMultiplier);
+    addProperty(_zoomOutBoundarySphereMultiplier);
```

Comment:
Not sure about this;

would it make sense to make this a Vec2Property instead with the "Range" view attached?

Reply:
wont fix